### DISCUSSION TITLE
Labels generator not showing item name or location

### DISCUSSION THREAD

**Discussion by floppy5-25** [sysadminsmedia/homebox#330] [Q&A] [answered]
Hello and thank you for continuing this project.
I am testing homebox before the gargantuan effort to actually inventory stuff but the label generator is not showing the item name or location. It just shows the QR code, asset id and two blank lines. 
It could be my install because the demo site does show text next to the QR codes. 
Can you please help me troubleshoot? 
Thanks.

Edit: Created a second test item and now the label generator shows item text next to the bar codes but with wrong asset id (put it another way, to the label generator, asset id 001-002 is 001-001 and 001-001 does not seem to exist) Could this be related to:
https://github.com/sysadminsmedia/homebox/issues/289#issue-2592804263

**Comment by tankerkiller125** [comment, marked as answer]
This is most likely related to #289
